### PR TITLE
Fix decode in ws_client

### PR DESCRIPTION
**Pull request by pigletfly**
fix https://github.com/kubernetes-client/python-base/issues/192

### CODE REVIEW

**Comment by palnabarun** [comment]
@pigletfly Can you please rebase against `master` to resolve the conflicts? :slightly_smiling_face:

**Comment by pigletfly** [comment]
@palnabarun rebased

File: stream/ws_client.py [hunk 1]
```diff
@@ -173,7 +173,8 @@ def update(self, timeout=0):
             elif op_code == ABNF.OPCODE_BINARY or op_code == ABNF.OPCODE_TEXT:
                 data = frame.data
                 if six.PY3:
-                    data = data.decode("utf-8", "replace")
+                    if op_code == ABNF.OPCODE_TEXT:
+                        data = data.decode("utf-8", "replace")
```
Comment (iciclespider) [inline stream/ws_client.py:177]: FYI, `ABNF.OPCODE_TEXT` will never be sent from kubernetes. All channels either send or receive `ABNF.OPCODE_BINARY` on the server side.

Comment (roycaihw) [reply stream/ws_client.py:177]: I think we need a test to demonstrate if this fix works.

**Pull request closed without merging**